### DISCUSSION TITLE
A variant of 00A775 LATIN SMALL LETTER RUM

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#255] [Q&A]
Enclosed please find a fragment of https://mufi.info/q.php?p=mufi/chars/unichar/42869.
![Screenshot from 2023-11-21 11-42-00_rum](https://github.com/psb1558/Junicode-font/assets/8024702/96438eab-9658-41ae-8122-e8cc26b5d464)

I had problem to find it in the documentation, but I looked into the wrong place :-( .

But am I right the form below is not available?
![latin_small_letter_rum](https://github.com/psb1558/Junicode-font/assets/8024702/c50da807-444c-428c-9b3d-b6bc97c8cb11)
It's OK with me, but I'm curious about the reason.
I understand the same situation is with DUM, LUM etc.

**Comment by psb1558** [comment]
I think (but am not sure) that the glyph used to have a short bar across the arm of the r, but I changed it for stylistic consistency with these guys:
<img width="347" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/53618bc2-a442-4ca4-9462-39dff12ba848">
But in the italic there was less need for change:
<img width="476" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/a143a9a6-c2f0-4a2c-b598-2068f29acfd3">
I saw no reason to keep the old roman form as a variant.

**Reply by psb1558** [reply]
This is U+A775 in version 1.003:
<img width="66" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/f24ae142-8053-4631-84ef-95883705fe2d">
Actually, I see that all the glyphs I cite above had short bars in version 1.003, so the change must have been for consistency with other glyphs or with other fonts. Unfortunately, I have only recently started to keep notes about revisions I make to this font.

**Reply by jsbien** [reply]
Thanks! "Proposal to add medievalist characters to the UCS" shows only short bars version, but from a 20th century book. In Polish 16th century print I see only the long bar version.

What do you think about these glyphs:
![Ungler2_pismo16_tab359-2-22_g+IS](https://github.com/psb1558/Junicode-font/assets/8024702/59cad91f-c28a-4bd7-8fa6-6cf374936211) ![snippet_Modus_page86x238y1033cum](https://github.com/psb1558/Junicode-font/assets/8024702/8fcd2d7c-7891-49ac-9b9d-14cc78518344)
![snippet_Modus_page18x46y995](https://github.com/psb1558/Junicode-font/assets/8024702/74a5c9b3-01f7-4e68-b412-e6f0851719bd)

Do they belong to the -um series or are something different?

**Reply by psb1558** [reply]
Can you supply a few lines of context for both of these characters—or a link to the book(s)? I suspect, though, from what I can see, that the second character may be a ci ligature. But I've never seen it before, and I can find nothing like it in Cappelli.

**Reply by jsbien** [reply]
Thanks for you interest.
As for the first one, it is only listed in a 20th century publication about 16th century Ungler's printing house, I never have seen it in text:
![Ungler2_pismo16_tab359](https://github.com/psb1558/Junicode-font/assets/8024702/602fd40b-fb3f-4515-8d7d-6d3d49e0d703)
As for the second one, it is used, but rarely, in _Modus legendi abreviaturas in utroque iure_, 1531.
http://sbc.org.pl/Content/35328. I enclose the snippets of all the occurrences I was able to find.
![snippet_Modus_page17x584y694](https://github.com/psb1558/Junicode-font/assets/8024702/848505fd-f7f4-4e93-baad-2b08a0fd3971)
![snippet_Modus_page96x78y1026](https://github.com/psb1558/Junicode-font/assets/8024702/c5ecc122-fbc1-4ab1-88b6-40b881b84647)

**Reply by psb1558** [reply]
This is useful, though. The c-form is pretty clearly an abbreviation for *cis*. The g-form is similar, but I'm afraid to guess at it. Thanks for the link to the book, which looks interesting: I'll look through it tomorrow. I see that Polona has a number of Ungler's prints: they'll be interesting to look through.

**Reply by jsbien** [reply]
Thanks! I have an incomplete list of digitized Ungler's prints. I can send you by mail if you wish me to.

**Reply by psb1558** [reply]
Thanks, I would like that.

**Reply by jsbien** [reply]
Sent on 22th.

**Comment by psb1558** [comment]
Sorry. It's been a long holiday weekend here, and I haven't had time to do a lot. I did notice this
<img width="925" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/fb17adf8-7d2b-4576-a35b-c54dbbbe6370">
in [Polona](https://polona.pl/item-view/e3e7f0a7-a30e-4164-b6c6-312c4dee848d?page=10) (for *ecclesiasticis*?), which leads me to wonder if your c-based character is a ligature with **is** U+A76D.

**Reply by psb1558** [reply]
I'm looking forward to going through the material you've sent.

**Reply by jsbien** [reply]
This seems a plausible interpretation.

**Comment by psb1558** [comment]
BTW, I've just uploaded a little package for LuaLaTeX and XeLaTeX users. I'd be very interested in your impressions/suggestions.

**Reply by jsbien** [reply]
Thanks for info. I will have a look at it probably next week.
